aws/resource_aws_ecs_service_test.go

```diff
@@ -1261,6 +1261,7 @@ resource "aws_ecs_service" "main" {
 	network_configuration {
 		security_groups = ["${aws_security_group.allow_all_a.id}", "${aws_security_group.allow_all_b.id}"]
 		subnets = ["${aws_subnet.main.*.id}"]
+		assign_public_ip = "ENABLED"
```

Comment:
I feel like the test needs something more but I am not sure what else to do besides this....